be/src/olap/rowset/rowset_writer_context.h

```diff
@@ -97,6 +97,7 @@ struct RowsetWriterContext {
     bool write_file_cache = false;
     bool is_hot_data = false;
     uint64_t file_cache_ttl_sec = 0;
+    uint64_t approximate_write_bytes = 0;
```

Comment:
suggest name approximate output bytes

Reply:
> approximate_bytes_to_write

i think `approximate_bytes_to_write` is better